schema/FIX43.xml

```diff
@@ -2357,7 +2357,7 @@
   </field>
   <field number='30' name='LastMkt' type='EXCHANGE' />
   <field number='31' name='LastPx' type='PRICE' />
-  <field number='32' name='LastQty' type='QTY' />
+  <field number='32' name='LastQty' type='INT' />
```

Comment:
These schema files are not intended to be edited in place. The idea is to copy them and use them as a template for your own custom schema. Your changes to these fields now make them non-standard.